src/components/Global/AccountItem.tsx

```diff
@@ -0,0 +1,116 @@
+import {Image, StyleProp, Text, View, ViewStyle} from "react-native";
+import { animPapillon } from "@/utils/ui/animations";
+
+import Reanimated, {
+  FadeOut,
+  ZoomIn
+} from "react-native-reanimated";
+import {PrimaryAccount, AccountService} from "@/stores/account/types";
+import {defaultProfilePicture} from "@/utils/ui/default-profile-picture";
+import {Check} from "lucide-react-native";
+import React from "react";
+import {useTheme} from "@react-navigation/native";
+
+const AccountItem: React.FC<{ account: PrimaryAccount, additionalStyles?: StyleProp<ViewStyle>, endCheckMark: boolean }> = ({
+  account,
+  additionalStyles,
+  endCheckMark
+}) => {
+  const theme = useTheme();
+  return (
+    <View
+      style={additionalStyles ? {
+        // @ts-expect-error
+        ...additionalStyles,
+        flexDirection: "row",
+        padding: 9,
+        borderStyle: "solid",
+        alignItems: "center",
+      }: {
+        flexDirection: "row",
+        padding: 9,
+        borderStyle: "solid",
+        alignItems: "center",
+      }}
+    >
```

Comment:
Peut-être ça pour simplifier ?

```suggestion
    <View
      style={{
        // @ts-expect-error
        ...(additionalStyles ?? {}),
        flexDirection: "row",
        padding: 9,
        borderStyle: "solid",
        alignItems: "center",
      }}
    >
```